### PR TITLE
Add Dockerfile with Bikeshed installed

### PR DESCRIPTION
**Pull request by emlun**
As noted in #936. The Dockerfile builds an image with the latest version of Bikeshed installed, to simplify local compilation of the spec. See the included README for usage instructions.

### CODE REVIEW

File: docker/bikeshed/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Docker image with `bikeshed`
+===
+
+Build the image:
+
+    alice@work $ docker build -t bikeshed docker/bikeshed --no-cache
```
Comment (equalsJeffH) [inline docker/bikeshed/README.md]: this presumes that one has docker installed on one's machine, yes?  we should say so if so.

Comment (emlun) [reply docker/bikeshed/README.md]: Yes; ok.

File: docker/bikeshed/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Then run it as `bikeshed` or `bikeshed watch`:
+
+- Either in one-shot mode:
+
+  ```
+  alice@work $ docker run --name bikeshed-webauthn -it --volume $(pwd):/spec bikeshed
```
[the remark this answers is not in the document]
Comment (emlun) [reply docker/bikeshed/README.md]: Done!

File: docker/bikeshed/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  alice@work $ docker start --attach bikeshed-webauthn
+  alice@work $ $BROWSER index.html
+  ```
+
+- Or in continuous watch mode:
+
+  ```
+  alice@work $ docker run --name bikeshed-webauthn-watch -it --volume $(pwd):/spec bikeshed watch
```
[the remark this answers is not in the document]
Comment (emlun) [reply docker/bikeshed/README.md]: Done!

**Comment by nadalin** [comment]
@emlun Please merge

**Pull request merged**